docs/assumptions.md

```diff
@@ -37,7 +32,7 @@ This can be useful, for example, to iterate DOM collections in older browsers.
 
 <div is="assumption-repl" data-assumption="arrayLikeIsIterable" data-plugins="transform-destructuring,transform-spread,transform-for-of">
 
-```text assumption-input
+```js
```

Comment:
We can re enable `js` markup now that we retrieve input code from `.token-line`.